setup.py

```diff
@@ -24,8 +24,9 @@ def __init__(self, user=False):
             import pybind11
             pybind11
         except ImportError:
-            if call([sys.executable, '-m', 'pip', 'install', 'pybind11']):
-                raise RuntimeError('pybind11 install failed.')
+            cmd = run([sys.executable, '-m', 'pip', 'install', 'pybind11'], capture_output=True)
```

Comment:
Ah... `capture_output=True` was added in 3.7 (but CI seems to still use 3.6). Py 3.6 was [end-of-life'd a year ago](https://endoflife.date/python) (and 3.7 in ~5 months) - would you be ok dropping that?